tests/pytests/test_voyager_drivers.py

```diff
@@ -81,12 +83,10 @@ def test_kernels():
 
 @pytest.mark.parametrize("label_type", ['isis3'])
 @pytest.mark.parametrize("formatter", ['isis'])
-@pytest.mark.skip(reason="Fails due to angular velocity problems")
 def test_voyager_load(test_kernels, label_type, formatter, isis_compare_dict):
     label_file = get_image_label('c2065022', label_type)
 
     usgscsm_isd_str = ale.loads(label_file, props={'kernels': test_kernels}, formatter=formatter)
     usgscsm_isd_obj = json.loads(usgscsm_isd_str)
     print(usgscsm_isd_obj)
```

Comment:
Left over print again. Probably from my PR :grimacing:

Reply:
Same as above, I actually like printing out the whole ISD on error.